src/docs/Core-2.0/TranslationAndLanguage/MigrationFromZLanguage.md

```diff
@@ -0,0 +1,31 @@
+Replacing ZLanguage
+===================
+
+In the examples below, the provided variable (e.g. $container, $kernel, etc) must be adjusted to match the environment
+from which it may be obtained. These variables are not defined explicitly in most cases.
+
+
+ZLanguage::getLanguageCode() becomes $container->getParameter('locale')
```

Comment:
Just saw `getLocale()` below.

What is the difference between `ZLanguage::getLanguageCode()` and `ZLanguage::getLocale()`?
What is the difference between `$container->getParameter('locale')` and `$request->getLocale()`?

The first is the default and the second one the current request's value only?